docs/resource/k8s.md

```diff
@@ -91,6 +91,7 @@ containers on your cluster.
 <!-- semconv k8s.pod -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
+| `k8s.pod.labels.<key>` | string | The labels placed on the Pod, the `<key>` being the label name, the value being the label value. | `pod.labels.app=my-app,pod.labels.mycompany.io/arch=x64` | Recommended |
```

Comment:
Prefixing everything with `k8s.pod.labels` is going to lead to some mixing of formats, is that acceptable?

For comparison, we have a [feature in the helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/867) that allows users to extract labels/attributes using the k8sattribute processor and we're not adding a prefix.  So far users have found it very natural/expected to see the labels themselves on the data instead of the label mixed with a semantic convention.

Reply:
It makes sense to me to have it prefixed (`k8s.pod.labels.*`) in order to be explicit and avoid potential conflicts given the key's cardinality. 

Also in the future we could add other resources' labels like `k8s.namespace.labels.*` and makes sense to have them separated in order to have the option for scoped searches.

Reply:
For recommended kubernetes label, we'll end up scoping twice.  Recommend labels like `app.kubernetes.io/instance: "mysql-abcxzy"` become `k8s.pod.labels.app.kubernetes.io/instance: "mysql-abcxzy"`. It feels strange to scope to `k8s`/`kubernetes` twice.

Reply:
Ah ok I see the point. However I'm not sure if we can just compromise the SemConv schema just relying on the recommended labels since users could potentially set any random label which could even lead to "conflicts".

To my eyes it would be safer to have the scoped prefix anyways even if in some cases the name would include the term k8s more than once.

Note that most probably "projecting" the labels directly to root level is not supported today in the Sem Conv.

Reply:
I was concerned about conflicts as well. The label name can be anything.

Reply:
@TylerHelmuth I'm new to this, but from others' comments looks like we must scope this way. If you see a better way please let me know and we'll discuss further.